boards/components/src/proximity.rs

```diff
@@ -38,9 +38,9 @@ impl<P: hil::sensors::ProximityDriver<'static>> ProximityComponent<P> {
         driver_num: usize,
     ) -> ProximityComponent<P> {
         ProximityComponent {
+            sensor,
             board_kernel,
             driver_num,
```

Comment:
Now I'm just confused — it's not even a consistent change? Here it leaves the `,`?